docs/docs/developer/_category_.json

```diff
@@ -1,4 +1,4 @@
 {
   "label": "Developer guide",
```

Comment:
Should we rename it? Maybe you've planned it for future steps with other changes we have discussed. But we need to clarify contributing VS extending the CRM which isn't the same